### DISCUSSION TITLE
AI教學網站 還不錯喔

### DISCUSSION THREAD

**Discussion by NcuJerryKao** [1101-GS3073-C/Python-Community-GS3073-C#487] [General]
此網站 內涵多個主題 且都有完整架構;化繁為簡 蠻易懂的 連我都學得會
https://sites.google.com/view/yhkao/ai-%E4%BA%BA%E5%B7%A5%E6%99%BA%E6%85%A7?authuser=0

**Comment by lx3211** [comment]
這個網站真的整理得很清楚明白耶
還有很多額外的網路參考資料

**Comment by manipulateed** [comment]
這個教材很適合新手踏入人工智慧領域，像這類的教學網站其實也還有很多，大家可以找到自己喜歡的風格再去做學習

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by williamyen0623** [comment]
這個網站看起來很簡潔，內容清晰，又是免費資源，很適合我這種窮學生，謝謝分享。

**Comment by 32844583** [comment]
不過我覺得影片內的老師講話有點平鋪直述，推薦去看李弘毅教授的影片，再搭配其他實作影片，這樣原理跟實作的方法就融會貫通了!
李弘毅: https://www.youtube.com/@HungyiLeeNTU
pytorch:https://www.youtube.com/watch?v=c36lUUr864M

**Comment by BingRong1217** [comment]
現在AI的教學網站越來越多了

**Comment by limitx0** [comment]
這網站的文章好完整
之後可以來看看

**Comment by awehao** [comment]
機概老師的網站诶

**Comment by RogelioKG** [comment]
看起來整理了很多實用的教材和影片，
先卡位存下來了👍。
